### PR TITLE
feat(): add ability to subscribe to space notifications

### LINKED ISSUE #708: Add browser notifications

### PR DESCRIPTION
**Pull request by midgerate**
Fixes #708 

- Currently it only adds the ability to save the users to our database which will be later used to send notifications. 

Depends on snapshot-hub https://github.com/snapshot-labs/snapshot-hub/pull/134

### CODE REVIEW

File: src/components/Block/Space.vue [hunk 1]
```diff
@@ -37,7 +56,22 @@ const isAdmin = computed(() => {
         <div class="mb-[12px] text-color">
           {{ $tc('members', nbrMembers, { count: _n(nbrMembers) }) }}
         </div>
-        <FollowButton :space="space" />
+        <div class="flex justify-center gap-x-2">
+          <FollowButton :space="space" />
+          <UiSidebarButton
```
Comment (samuveth) [inline src/components/Block/Space.vue:61]: Maybe we could rename component to `CircleButton` as this is being used in other places too now.

File: src/composables/useSpaceSubscription.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      loading.value = false;
+    }
+  }
+
+  async function toggleSubscription() {
+    if (web3.value.authLoading) {
+      return null;
+    }
+
+    if (!web3Account.value) {
```
Comment (samuveth) [inline src/composables/useSpaceSubscription.ts:58]: the subscription button isn't visible when there is not `web3Account`, can we remove this check?

**Comment by bonustrack** [comment]
@midgerate The PR for the hub is merged but I'm still not able to subscribe for a space, it throw this error `Error: Unexpected error value: "request failed"`, can you try make it work?

File: src/components/Block/Space.vue [hunk 3]
```diff
@@ -1,9 +1,10 @@
 <script setup>
-import { computed } from 'vue';
+import { computed, ref, watchEffect } from 'vue';
 import { getInstance } from '@snapshot-labs/lock/plugins/vue3';
 import { useWeb3 } from '@/composables/useWeb3';
 import { useApp } from '@/composables/useApp';
-
+import { useSpaceSubscription } from '@/composables/useSpaceSubscription';
+import { useFollowSpace } from '@/composables/useFollowSpace';
```
Comment (bonustrack) [inline src/components/Block/Space.vue:7]: We should separate import and code with an empty line
```suggestion
import { useFollowSpace } from '@/composables/useFollowSpace';

```

**Review by bonustrack — APPROVED** [review]
Looks good

**Pull request merged**